canonical-web-and-design/charmhub.io
# Implement the new navigation

## Done
- _Implement the new navigation._

## How to QA
- Visit https://charmhub-io-941.demos.haus
- See the navigation looks and behaves the same as the one on http://juju.is/

## Issue / Card
Fixes https://github.com/canonical-web-and-design/snap-squad/issues/1997

## Screenshots
